File: Data Structures & Algorithms/two-integer-sum/submission-2.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        ans = []
        for i in range(len(nums)):
            for j in range(len(nums)):
                if nums[i] + nums[j] == target and i != j:
                    if i < j:
                        ans.append(i)
                        ans.append(j)
                        return ans
                    else:
                        ans.append(j)
                        ans.append(i)
                        return ans

        

File: Data Structures & Algorithms/two-integer-sum-ii/submission-2.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int[] answer = new int[2];
        for (int i = 0; i<numbers.length ; i++){
            for (int j = 1; j<numbers.length ; j++){
                if (numbers[i] + numbers[j] == target && i<j){
                    answer[0] = i+1;
                    answer[1] = j+1;
                }
            }
        }
        return answer;
    }
}
